### PR TITLE
ok. I cleaned up. the -reportWebVitals.js- is still a question how to…

### PR DESCRIPTION
**Pull request by mBraun1970**
… get it in the no need folder without error .....

### CODE REVIEW

File: src/AlbumCard/AlbumCard.css [hunk 1]
```diff
@@ -14,3 +18,18 @@
...
+
+.album-list {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.AlbumCard img {
+  max-width: 100vw;
+  height: 50vh;
```
Comment (F-Kirchhoff) [inline src/AlbumCard/AlbumCard.css:31]: please don't use vh and vw here, you can use width: 100% and "aspect-ratio: 1 / 2"

File: src/AlbumCard/AlbumCard.js [hunk 2]
```diff
@@ -4,10 +4,22 @@ console.log("in AlbumCard ---> 1");
...
-    </div>
+
+    // this is the solution , let me get there :) check from line 18
+    <section className="album-card">
+      <img src={image.url} alt={title} className="album-card__image" />
+      <aside className="album-card__info">
+        <h2>{title}</h2>
+        <small>{artist}</small>
+      </aside>
+    </section>
```
Comment (F-Kirchhoff) [inline src/AlbumCard/AlbumCard.js:15]: very nice semantic html!

File: src/AlbumCard/AlbumCard.js [hunk 3]
```diff
@@ -4,10 +4,22 @@ console.log("in AlbumCard ---> 1");
 
 export function AlbumCard({ title, artist, image }) {
   return (
-    <div className="album-card">
-      <article className="albumTitle">Title: {title}</article>
-      <article className="albumArtist">Artist: {artist}</article>
-      <article className="albumImage">Immmmage: {image}</article>
-    </div>
+
+    // this is the solution , let me get there :) check from line 18
```
Comment (F-Kirchhoff) [inline src/AlbumCard/AlbumCard.js:8]: What do you mean by this comment?

File: src/AlbumCard/AlbumCard.js [hunk 4]
```diff
@@ -4,10 +4,22 @@ console.log("in AlbumCard ---> 1");
...
+        <small>{artist}</small>
+      </aside>
+    </section>
+
+
+    // <div className="album-card">
+    //   <article className="albumTitle">Title: {title}</article>
+    //   <article className="albumArtist">Artist: {artist}</article>
+    //   <article className="albumImage">Image: {image}</article>
+    // </div>
```
Comment (F-Kirchhoff) [inline src/AlbumCard/AlbumCard.js:22]: please remove any comments like this before merging.

File: src/App.css [hunk 5]
```diff
@@ -2,37 +2,16 @@
...
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
+  min-width: 60vw;
+  /* display: flex; */
+  /* flex-direction: column; */
+  /* align-items: center; */
+  /* justify-content: center; */
```
Comment (F-Kirchhoff) [inline src/App.css:12]: please remove those comments as well.

File: src/App.js [hunk 6]
```diff
@@ -1,26 +1,31 @@
...
-        <header className="AppTitle">Music Collector</header>
-
-        <section className="AlbumCard">
-          get <br></br>
-          the AlbumCard <br></br>here -
-          {data.map((album, index) => (
-            <AlbumCard key={index} title={album.title} artist={album.artist} />
+      <section className="AlbumCard">
+        <br></br>check below <br></br>
+        the AlbumCards
```
Comment (F-Kirchhoff) [inline src/App.js:15]: Please remove this text.

File: src/App.js [hunk 7]
```diff
@@ -1,26 +1,31 @@
...
+        <article className="album-list">
```
Comment (F-Kirchhoff) [inline src/App.js:16]: your album list should be a "ul"

File: src/App.js [hunk 8]
```diff
@@ -1,26 +1,31 @@
...
+          {data.map((album) => (
+            <aside key={album.id}>
```
Comment (F-Kirchhoff) [inline src/App.js:18]: this should be a 'li'

**Review by F-Kirchhoff — CHANGES_REQUESTED** [review]
This looks great already! Please have a look, I found some minor issues with your html elements.

File: src/App.js (hunk 8 above)
Comment (mBraun1970) [reply src/App.js:16]: first you say nice semantic , now i get a change ... :) .... ok

File: src/App.js (hunk 7 above)
Comment (mBraun1970) [reply src/App.js:14]: done

File: src/AlbumCard/AlbumCard.js (hunk 2 above)
Comment (mBraun1970) [reply src/AlbumCard/AlbumCard.js:13]: juhu :)

File: src/AlbumCard/AlbumCard.js (hunk 3 above)
Comment (mBraun1970) [reply src/AlbumCard/AlbumCard.js]: this was a left over , its gone now

**Review by F-Kirchhoff — CHANGES_REQUESTED** [review]
you still need to change the html elements to ul and li.

File: src/App.js [hunk 9]
```diff
@@ -1,26 +1,30 @@
...
-      <main className="App-header">
-        <header className="AppTitle">Music Collector</header>
-
-        <section className="AlbumCard">
-          get <br></br>
-          the AlbumCard <br></br>here -
-          {data.map((album, index) => (
-            <AlbumCard key={index} title={album.title} artist={album.artist} />
+      <section className="AlbumCard">
+        <article className="albumList">
```
Comment (onemanwenttomow) [inline src/App.js:15]: I think this should be a <ul> element.

**Review by onemanwenttomow — APPROVED** [review]
One element to change otherwise it looks good to me!

**Pull request merged**